api/v1alpha1/inferencepool_types.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// LLMServerPoolSpec defines the desired state of LLMServerPool
-type LLMServerPoolSpec struct {
+// InferencePoolSpec defines the desired state of InferencePool
+type InferencePoolSpec struct {
 
 	// ModelServerSelector uses a map of label to watch model server pods
```

Comment:
Why isn't this just "Selector"? The "ModelServer" prefix feels unnecessarily repetitive

Reply:
Changed! However, I don't think we use ModelServer anywhere else in the API, and I'm a tad worried `selector` is somewhat obtuse. But I don't feel overly strong here.